docs/content/en/docs/administration/permissions/put_declarative_organziation_permissions.md

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
There is a typo in the name of the file, please update to `put_declarative_organization_permissions`

Reply:
@JKlucka This would be the last change :)

Reply:
Omg, sorry for that! Thank you

Reply:
Done @Mara3l